templates/test.template

```diff
@@ -2,21 +2,20 @@
 // This code is governed by the BSD license found in the LICENSE file.
 
 /*---
-template: default
 author: Mathias Bynens
-desc: Unicode property escapes `\p{<%= UnicodePropertyValueExpression %>}` and `\P{<%= UnicodePropertyValueExpression %>}`
+description: >
+  Unicode property escapes `\p{}` and \P{}` for `<%= UnicodePropertyValueExpression %>`
 esid: sec-static-semantics-unicodematchproperty-p
-flags: [generated]
```

Comment:
another suggestion, add an info tag to tell which unicode version is being used. This helps as a good reference. E.g:

```yaml
info: |
  Unicode version: <%= UnicodeVersion %>
```